src/applications/vaos/tests/new-appointment/redux/helpers/formSubmitTransformers.unit.spec.js

```diff
@@ -1,826 +0,0 @@
-import { expect } from 'chai';
-
-import {
-  transformFormToCCRequest,
-  transformFormToVARequest,
-  transformFormToAppointment,
-} from '../../../../new-appointment/redux/helpers/formSubmitTransformers';
-import { FETCH_STATUS, VHA_FHIR_ID } from '../../../../utils/constants';
-
-describe('VAOS data transformation', () => {
```

Comment:
It might be worth keeping these tests, since I'm not sure how comprehensively we test all the data variations and specific fields when submitting data. Unless you found this was fully covered elsewhere.